lib/generators/manageiq/plugin/templates/.github/workflows/ci.yaml

```diff
@@ -0,0 +1,47 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  schedule:
+  - cron: '0 0 * * *'
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version:
+        - '2.6'
+        - '2.7'
```

Comment:
Can this be auto-generated similar to how it was done in the .travis.yml?

```erb
<%= YAML.load_file(Rails.root.join(".travis.yml"))["rvm"].map { |r| "- #{r}" }.join("\n") %>
```

Reply:
We could but I thought this was a little strange, maybe I'm missing something but wouldn't we just update this list when we update the list in core's `.github/workflows/ci.yaml`?  Not seeing why this needs to be built dynamically

Reply:
> We could but I thought this was a little strange, maybe I'm missing something but wouldn't we just update this list when we update the list in core's .github/workflows/ci.yaml

It's easy to forget the template.  This way, it's automatic and we can't forget.

Reply:
Still seems overly complicated but if you don't want to merge without it I'll get back to this in a bit